next.config.js

```diff
@@ -29,7 +29,7 @@ module.exports = withMdxEnhanced({
               `frame-ancestors 'none'`,
               `worker-src 'self' blob:`,
               `child-src 'self' blob:`,
-              `connect-src 'self' ${process.env.NEXT_PUBLIC_TOKEN_API_URL} ${process.env.NEXT_PUBLIC_SUPABASE_URL} https://api.github.com https://api.mapbox.com https://events.mapbox.com`,
+              `connect-src 'self' ${process.env.NEXT_PUBLIC_TOKEN_API_URL} ${process.env.NEXT_PUBLIC_SUPABASE_URL}  ${process.env.NEXT_PUBLIC_MATOMO_URL} https://api.github.com https://api.mapbox.com https://events.mapbox.com`,
```

Comment:
Yes, that is indeed important to make it work. I've tried building the project locally to test if the CSP is working, but it told me that `${process.env.NEXT_PUBLIC_MATOMO_URL}` needs to also be added in the `img-src` directive of the CSP (which makes sense since we are doing image-based tracking).